packages/react-native/lib/index.js

```diff
@@ -92,42 +96,46 @@ export function useAccount(address) {
   };
 }
 
-function getStorageDocuments() {
-  return AsyncStorage.getItem('dock-wallet')
-    .then(data => {
-      if (!data) {
-        return [];
-      }
-
-      data = JSON.parse(data);
-
-      return Object.keys(data).map(key => data[key]);
-    })
-    .catch(() => []);
-}
-
 export function useWallet({syncDocs = true} = {}) {
   return useContext(WalletSDKContext);
 }
 
-export function WalletSDKProvider({onError, customUri, children, onReady}) {
+export function _useWalletController() {
   const [wallet, setWallet] = useState();
   const [status, setStatus] = useState('loading');
+  const [testMode, setTestMode] = useState(false);
   const [documents, setDocuments] = useState([]);
-  const webViewRef = useRef();
-  const sandboxWebViewRef = useRef();
-  const baseUrl =
-    Platform.OS === 'ios' ? 'app-html' : 'file:///android_asset/app-html';
+  const [firstFetch, setFirstFetch] = useState();
+  const networkId = useMemo(() => {
+    return wallet?.getNetworkId();
+  }, [wallet]);
+
+  console.log('Current networkId', networkId);
+  console.log('wallet documents', documents);
```

Comment:
we probably don't want to dump all the documents out - that could be pretty large, right? and possibly hold PII

Reply:
Fixed